test_utils/factories.py

```diff
@@ -46,6 +46,7 @@ class Meta(object):
     name = factory.LazyAttribute(lambda x: FAKER.company())
     active = True
     site = factory.SubFactory(SiteFactory)
+    identity_provider = factory.LazyAttribute(lambda x: FAKER.slug())
```

Comment:
@mattdrayer fuzzy is considered deprecated: http://factoryboy.readthedocs.io/en/latest/fuzzy.html and https://github.com/FactoryBoy/factory_boy/issues/271/